### PR TITLE
Html And CSS branch

### PR DESCRIPTION
**Pull request by tawakuliKH**
This branch includes html and css files for the PR

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+                        <li class="tag"><a href="">html</a></li>
+                    </div>
+                    <div>
+                        <li class="tag"><a href="">CSS</a></li>
+                    </div>
+                    <div>
+                        <li class="tag"><a href="">Javascript</a></li>
+                    </div>
+                </div>
+
```
Comment (tawakuliKH) [inline index.html:79]: - [ ] Kindly use li tags in a nav tag, The nav tag is for semantics reasons and the right position for convention and user experience for mobile devices.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+            <ul class="social_nav">
+                <li><a href="twitter.com/tawakulikhadim"><img src="assets/image_social/vitctor_t.svg" alt="Twitter"></a>
+                </li>
+                <li><a href="https://github.com/tawakuliKH"><img src="assets/image_social/victor_g.svg"
+                            alt="Github"></a></li>
+                <li><a href=""><img src="assets/image_social/victor_h.svg" alt=""></a></li>
+                <li><a href=""><img src="assets/image_social/victor_m.svg" alt="Mediam"></a></li>
+                <li><a href=""><img src="assets/image_social/victor_l.svg" alt="Linked In"></a></li>
+            </ul>
+
```
Comment (tawakuliKH) [inline index.html:37]: - [ ] Kindly remove extra spaces in basecode, extra spaces and newlinex makes your code not clean.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/tawakuliKH)
+- Twitter: [@twitterhandle](https://twitter.com/tawakulikhadim)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/khadim-tawakuli)
```
Comment (tawakuliKH) [inline README.md:167]: - [ ] Kindly write your social media profile link properly, people can reach you easily after viewing your works.

**Review by tawakuliKH — COMMENTED** [review]
**Highlits**
<hr>
God Job! :100: <br>
There is some Issues you need to fix but you are almost there!

170121142-a9f93bb7-38bb-45cc-8327-aab2501ef90a


**Top Higlights** 

- [ ]  Correct uses of branches :+1: 
- [ ]  Readme file is meaningful :+1: 
- [ ]  Commit message are clear :+1: 
- [ ] Linters error are handled :+1: 
- [ ] Good use of github flow :+1: 



**Optional Changes**
<hr>
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding! :clap: :clap: :clap: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Review by amejid — APPROVED** [review]
Hi @tawakuliKH, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Good use of Github features :heavy_check_mark: 
- Comments are polite and provide a reason :heavy_check_mark: 
## Optional suggestions
- It would be great to add some emojis to make the review more human-friendly :+1: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**